Modix.Bot/Modules/MarkerRoleModule.cs

```diff
@@ -16,9 +16,9 @@ namespace Modix.Modules
 {
     [Group("pingrole")]
     [Alias("pingroles")]
```

Comment:
Is this intended to be a `!help`-only change, or do we think there's value in changing the command and class name as well?

Reply:
I can open a separate PR that renames the appropriate classes and interfaces to match the new canonical term, if you'd like. A bigger question is what to do about the `!pingrole` command. Should we also add `!topicrole` as an alias?

Reply:
I suppose if I had to choose between changing associated code names, or not, changing makes slightly more sense. Feel free to submit a followup PR, I guess.